src/main/java/Duke/Tasks/Task.java

```diff
@@ -0,0 +1,41 @@
+package Duke.Tasks;
+
+public class Task {
+    public String description;
+    protected boolean isDone;
+
+    public Task(String description, int status) {
+        this.description = description;
+        if (status == 1) {
```

Comment:
Avoid using magic number such as `1`, consider extracting it out into a constant.